src/stripeLogTextDocumentContentProvider.ts

```diff
@@ -0,0 +1,39 @@
+import * as vscode from 'vscode';
```

Comment:
This file is copied from the one for events. There might be a way to share the logic with a base class or something.

Reply:
I think we can do this with one class too: 

1. Keep two schemas and register two instances of a `StripeResourceDocumentContentProvider`. The `StripeResourceDocumentContentProvider` takes in the regex pattern and a function to fetch the object from the extension context in its constructor.

or 

2. We have one schema, and one instance of a `StripeResourceDocumentContentProvider`. But that class can figure out on its own what type of resource the resourceId represents is and how to fetch it. Maybe it can have a mapping of the regex match to retrieve function, and it tries to loop through that mapping to see if any match the resourceId it was given.


WDTY? I think I'd vote for 1 just because 2 defers more logic to happen at runtime.